tensorflow/lite/examples/label_image/BUILD

```diff
@@ -62,6 +62,7 @@ cc_library(
         "//tensorflow/lite:framework",
         "//tensorflow/lite:string",
         "//tensorflow/lite:string_util",
+        "//tensorflow/lite/delegates/xnnpack:xnnpack_delegate",
```

Comment:
Since you are importing `xnnpack_delegate.h` in `label_image.cc`, we need this BUILD dependency in the `label_image` cc_binary RULE :-)

Reply:
Thanks. Done. Just curious, why indirect dependency doesn't work?

Reply:
Normally it does, but in some of optimized BUILD scenarios tested internally, it returns in failures.